modules/core/src/tools/time/vpTime.cpp

```diff
@@ -80,13 +86,16 @@ double minTimeForUsleepCall = 4;
 double getMinTimeForUsleepCall() { return minTimeForUsleepCall; }
 
 /*!
-  \fn vpTime::measureTimeMs()
-  Return the time in milliseconds since January 1st 1970.
+  Return the time in microseconds since January 1st 1970.
```

Comment:
Documentation states that it returns the time in microseconds since January 1st 1970.
It should not be true with `QueryPerformanceFrequency`, here [a quick test](https://rextester.com/XKOK1632).

With [`system_clock`](https://en.cppreference.com/w/cpp/chrono/system_clock) it should be better.